packages/auth/package.json

```diff
@@ -124,7 +124,7 @@
     "@rollup/plugin-json": "4.1.0",
     "@rollup/plugin-strip": "2.1.0",
     "@types/express": "4.17.17",
-    "chromedriver": "98.0.1",
+    "chromedriver": "114.0.2",
```

Comment:
This pins the version to 114.0.2 for manual installs/test runs on our development machines?

Reply:
Yes, of the chromedriver dependency which downloads a varying version of the chromedriver binary (depending oh the current version of chrome.)